src/runtime/analytics-service.js

```diff
@@ -19,25 +19,27 @@ import {createElement} from '../utils/dom';
 import {setImportantStyles} from '../utils/style';
 import {AnalyticsRequest,
         AnalyticsContext} from '../proto/api_messages';
+import {TransactionId} from './transaction-id';
+import {parseQueryString, parseUrl} from '../utils/url';
 
 /** @const {!Object<string, string>} */
 const iframeStyles = {
   display: 'none',
 };
 
+
 export class AnalyticsService {
   /**
-   * @param {!../model/doc.Doc} doc
-   * @param {!web-activities/activity-ports.ActivityPorts} activityPorts
-   * @param {!../model/page-config.PageConfig} config
+   * @param {!./deps.DepsDef} deps
```

Comment:
The naming is confusing for me. deps isn't very descriptive. When instantiating this you're passing a ConfiguredRuntime type.

Reply:
Runtime implements those APIs, and this is the pattern we follow for all flows.

Reply:
`ConfiguredRuntime` implements `Deps`, which is short for dependencies. But it's not configured runtime itself.